sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/jdbc/JdbcPropertiesBeanPostProcessor.java

```diff
@@ -49,6 +51,12 @@ class JdbcPropertiesBeanPostProcessor implements BeanPostProcessor, EnvironmentA
     private GenericApplicationContext applicationContext;
     private Environment environment;
 
+    @Override
+    public int getOrder() {
+        // Runs before JdbcConnectionDetailsBeanPostProcessor
+        return Ordered.HIGHEST_PRECEDENCE + 3;
```

Comment:
Does the `JdbcConnectionDetailsBeanPostProcessor` has an order value defined? maybe we should reference that?

Reply:
Yes.